mix.exs

```diff
@@ -36,8 +36,10 @@ defmodule ExMachina.Mixfile do
     [
       {:ex_doc, "~> 0.14", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
-      {:ecto, "~> 2.1", optional: true},
-      {:postgrex, ">= 0.0.0", only: [:test]},
+      {:ecto, "~> 2.2 or ~> 3.0", optional: true},
+      {:jason, "~> 1.0", optional: true},
+      {:ecto_sql, "~> 2.2 or ~> 3.0", only: :test},
```

Comment:
Should `ecto_sql` have an `optional: true` here since ex_machina can be used without ecto?

Reply:
It's limited to `only: test` so it should work the same way